piksi_tools/console/tracking_view.py

```diff
@@ -133,104 +133,123 @@ class TrackingView(CodeFiltered):
                 Item('legend_visible', label="Show Legend:"),
                 CodeFiltered.get_filter_group(), )))
 
+    def clean_cn0(self, t):
+        assert self.CN0_lock.locked()
+        for k in list(self.CN0_dict.keys()):
+            if self.CN0_age[k] < self.time[0]:
```

Comment:
If the age of the data in the CN0 dict is older than the "oldest active time" then delete the data so it can be cleared out of the plots and the plot data.